src/TraceEvent/TraceEvent.csproj

```diff
@@ -63,6 +63,7 @@
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Web" />
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
+    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="5.2.9" />
```

Comment:
Per https://www.nuget.org/packages/Microsoft.IdentityModel.Clients.ActiveDirectory/, this package is no longer maintained, and https://www.nuget.org/packages/Microsoft.Identity.Client/ is the recommended alternative.  Would you be able to convert to using the new recommended package?